### PR TITLE
Ipv4 netmask

### PR DESCRIPTION
**Pull request by hans-vm**

### CODE REVIEW

File: core/converter/validator/ipv4-with-netmask-validator.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ exports.IPv4WithNetmaskValidator = Validator.specialize({
                 var splitValue = value.split("/");
                 // There may only be one "/", no more and no less
                 if (splitValue.length === 2) {
-                    if (new IPv4Validator().validate(splitValue[0])
+                    if (new Ipv4Validator().validate(splitValue[0])
```
Comment (pchaussalet) [inline core/converter/validator/ipv4-with-netmask-validator.js]: You should avoid creating a new instance each time, it is a stateless object.

File: core/converter/validator/ipv4-netmask-validator.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            if (typeof netmask === 'string') {
+                // Subnet mask should be a valid IP address
+                if (new Ipv4Validator().validate(netmask)) {
+                    // Split by dots
+                    var d = netmask.split('.');
+                    // Convert to binary format
+                    var b = ((((((+d[0])*256)+(+d[1]))*256)+(+d[2]))*256+(+d[3])).toString(2);
+                    // Check if there's any `1` coming after `0`
+                    // Binary should be either 32-bits long or `0`
+                    // Maybe a better solution would be zero padding and check?
```
Comment (pchaussalet) [inline core/converter/validator/ipv4-netmask-validator.js]: Indeed, it sound easier (and more understandable) to me, you should do that change.

**Review by pchaussalet — CHANGES_REQUESTED** [review]

**Comment by hans-vm** [comment]
@pchaussalet Made changes. Please go through it again and let me know your feedback. Thanks!

**Review by pchaussalet — APPROVED** [review]
LGTM

**Review by pchaussalet — APPROVED** [review]
LGTM

**Pull request merged**